static/openapi/vwrs-spec.json

```diff
@@ -467,6 +467,70 @@
             "example": "test-service-us-east.example.io"
           }
         }
+      },
+      "GetStatusResponse": {
+        "type": "object",
+        "properties": {
+          "avg_waiting_time": {
+            "type": "number",
+            "format": "float",
+            "description": "The average time users spend in the virtual waiting room before being granted access to the origin service (in milliseconds).",
+            "example": 19871.8
+          },
+          "backoff_interval": {
+            "type": "integer",
+            "description": "Indicates the interval in seconds after which the EdgeWorker should call the status API call to the vwrs server. This is used to reduce server load.",
+            "example": 1
+          },
+          "position": {
+            "type": "integer",
+            "description": "The client's current position in the waiting room.",
```

Comment:
Are we calling them users or clients? I think we also use visitors somewhere. In this PR, we're using at least _client_ and _users_.